apps/PrusaSlicer/install-32.save

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I don't think you meant to create this file.

Reply:
Oh, no I didn't.. I force checked out a new branch, and I thought it would delete all the changed files, but I guess not..